dev-bin/release.sh

```diff
@@ -30,6 +30,7 @@ tag="v$version"
 
 rm -fr vendor
 
+perl -pi -e "s{(?<=php composer\.phar require maxmind\/minfraud:).+}{^$version}g" README.md
```

Comment:
`maxmind-db/reader` I think?

Reply:
Hmm, maybe I forgot to save the file as I copied it from the docs when I tested it by setting the version. Anyway, I force-pushed a fix. Thanks!